lib/seahorn/BvOpSem2.cc

```diff
@@ -2743,6 +2779,8 @@ Expr Bv2OpSem::errorFlag(const BasicBlock &BB) {
   return this->OperationalSemantics::errorFlag(BB);
 }
 
+Bv2OpSem::~Bv2OpSem() {}
```

Comment:
this definitely not needed. You might need `BvOpSem::~BvOpSem() = default;` but never empty